src/ert/gui/simulation/view/progress_widget.py

```diff
@@ -90,9 +89,16 @@ def repaint_components(self) -> None:
     def stop_waiting_progress_bar(self) -> None:
         self._waiting_progress_bar.setVisible(False)
 
+    def start_waiting_progress_bar(self) -> None:
```

Comment:
Is it reasonable to write a simple test for it? Ie. `self._wait_progress_bar active` or not?

Reply:
I am not sure; it hanging or not will most likely go unnoticed by the users anyways. I can add a small unit test, but I do not think we should put too much effort into it.

Reply:
Hm, I see. Currently it is not tested at all. Then I guess I'll leave it up to you :)